danielssonn/ethTO
# feat: add lend ui

## Purpose

1. Added a `LendCheckout` component
2. Added token images to prices

## Testing Instructions

1. Confirm the lend flow looks correct (optional):

- Pull down the branch
- Move into the `client` directory and run `npm run dev`
- Navigate to `http://localhost:3000/lend`
- Click on This would be your NFT (click here)

2. Review code

## Checklist
